platform_secrets/config.py

```diff
@@ -23,3 +24,4 @@ class Config:
     platform_auth: PlatformAuthConfig
     kube: KubeConfig
     cluster_name: str
+    events: Optional[EventsClientConfig]
```

Comment:
```suggestion
    events: EventsClientConfig | None
```

I guess that the modern syntax is acceptable here